Processing/VALL_Deep_Dive.Rmd

```diff
@@ -0,0 +1,404 @@
+---
+title: "VALL_Deep_Dive"
+author: "ROSSyndicate"
+date: "2024-11-12"
+output: html_document
+editor_options: 
+  chunk_output_type: console
+---
+
+This R markdown prepares data inputs and figures for the Deep Dive Climate Change Vulnerability Assessment for Water Supplies at Valles Caldera National Preserve.
+
+### 1. Set up workspace
+
+First, install and/or load required packages and functions.
+
+```{r setup, include = TRUE}
+knitr::opts_chunk$set(echo = TRUE)
+sf::sf_use_s2(FALSE)
```

Comment:
I think I've brought this up before, but is there a reason to keep adding this? I think it should be removed...s2 is a spherical geometry that is more accurate. However I believe the sf package will turn it on/off on its own based on whether the sf object is in projected or geographic coordinates.

Reply:
There are some functions that are still breaking without this explicitly written out. I still need to determine which exactly is causing the issue!